compiler_opt/rl/corpus.py

```diff
@@ -65,14 +103,17 @@ def build_modulespecs_from_datapath(
   module_specs: List[ModuleSpec] = []
 
   # This takes ~7s for 30k modules
-  for module_path in module_paths:
+  for rel_module_path in module_paths:
+    full_module_path = os.path.join(data_path, rel_module_path)
     exec_cmd = _load_and_parse_command(
-        module_path=os.path.join(data_path, module_path),
+        module_path=full_module_path,
         has_thinlto=has_thinlto,
         additional_flags=additional_flags,
         delete_flags=delete_flags,
         cmd_override=cmd_override)
-    module_specs.append(ModuleSpec(name=module_path, exec_cmd=tuple(exec_cmd)))
+    size = os.path.getsize(full_module_path + '.bc')
```

Comment:
does this change the estimate on line 105?

Reply:
Nope. But now that you've mentioned it the estimate on 105 does have the advantage of all the files being cached in RAM...

Reply:
you mean all the file paths? but so are the `ModuleSpec`s, no? 

On that - separately - we could look at making `ModuleSpec` use `__slots__` to reduce total memory footprint of a collection of these things.

Reply:
I meant that the file/directory is cached to RAM, so accesses were faster when I got that 7s/30k modules number, don't think the size of the ModuleSpecs would be concerning (for a single project's worth of ModuleSpecs)